src/_guides/libraries/c-interop.md

```diff
@@ -168,6 +168,41 @@ For details, see the following:
 * [Flutter dart:ffi page][binding]
 * [dart:ffi examples]({{page.samples}})
 
+## Interfacing with native types
+
+The dart:ffi library provides multiple types 
+which represent native types in C.
+
+They all implement [`NativeType`][] and
+serve purely as markers in type signatures.
+Subtypes of `NativeType` are not instantiable in Dart code.
+
+The following are some common types used as markers.
+Reference their API documentation for further subtypes. 
+
+| **Dart type**                                                                         | **Description**                                                  |
+|---------------------------------------------------------------------------------------|------------------------------------------------------------------|
+| [AbiSpecificInteger]({{site.dart_api}}/stable/dart-ffi/AbiSpecificInteger-class.html) | The supertype of all [Abi][] specific integer types.             |
```

Comment:
Dart 2.17 adds ABI-specific types such as Char, UnsignedChar, WChar, etc. to the list of predefined types.

I don't believe API-documentation for those has been generated yet, so that should probably be a follow up PR of this one.

Reply:
Yep, I plan to follow up with those additions in a separate PR which will land alongside 2.17. Thanks!